Exception/GoogleVideoException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * This file is part of the PrestaSitemapBundle package.
+ *
+ * (c) PrestaConcept <www.prestaconcept.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Presta\SitemapBundle\Exception;
+
+/**
+ * Exception used when limit is reached on adding video
+ *
+ * @author David Epely <depely@prestaconcept.net>
+ */
+class GoogleVideoException extends GoogleVideoUrlException
```

Comment:
The new exception should not inherit from the old one, it should be the opposite, like : https://github.com/twigphp/Twig/blob/2.x/lib/Twig/Environment.php

Reply:
I'm not agree with that, this example is IMHO not ok when we are talking about exceptions. The new code throws a `GoogleVideoException`, but the application may try to catch the old exception `GoogleVideoUrlException`. If `GoogleVideoUrlException` extends from `GoogleVideoException` and the new code throws a `GoogleVideoException`, exception won't be caught by the application. Are you ok with that ?

Reply:
I'm not agree with that, this example is IMHO not ok when we are talking about exceptions. The new code throws a `GoogleVideoException`, but the application may try to catch the old exception `GoogleVideoUrlException`. If `GoogleVideoUrlException` extends from `GoogleVideoException` and the new code throws a `GoogleVideoException`, exception won't be caught by the application. Are you ok with that ?

Reply:
OK. You convinced me. Let's try it like that.